pkg/config/template.go

```diff
@@ -882,6 +887,12 @@ const templateStringCrioRuntimeAdditionalDevices = `# List of additional devices
 
 `
 
+const templateStringCrioRuntimeCDISpecDirs = `# List of directories to scan for CDI Spec files.
+cdi_spec_dirs = [
+{{ range $cdiSpecDirs := .CDISpecDirs }}{{ printf "\t%q,\n" $cdiSpecDirs}}{{ end }}]
```

Comment:
after https://github.com/cri-o/cri-o/pull/5007 we have changed the structure of these to have `{{ $.Comment }}` PTAL at the other slice config fields for an example

Reply:
Updated to follow those conventions.